docs/README.md

```diff
@@ -0,0 +1,20 @@
+# Docs
+
+`docs/` contains end user documentation geared towards writing smart contracts with PyTeal.  The README explains how to edit docs.
+
+## Local testing
+
+Pyteal uses [Sphinx](https://github.com/sphinx-doc/sphinx) to generate HTML from RST files.  Test changes by generating HTML and viewing with a web browser.
+
+Here's the process:
+* Perform one-time environment setup:
+  * Create a virtual environment isolated to `docs/`.  Do _not_ mix with a top-level virtual environment.
+  * Install dependencies via `pip`.
+* Generate HTML docs:  `make html`.
+* If successful, generated HTML is available in `docs/_build/html`.
```

Comment:
Does that still mean we are using *only-one top-level venv*?

Can also make dependencies installation more detailed by `pip install -r requirements.txt`.

Reply:
@ahangsu My thoughts are inline below.  If it looks good, can you resolve?

> Does that still mean we are using only-one top-level venv?

Yes.  Though I assume it's a user choice if he/she _wants_ a separate venv isolated to docs.  There's no technical reason preventing the user from sharing the venv.

> Can also make dependencies installation more detailed by pip install -r requirements.txt.

I took the feedback in via https://github.com/algorand/pyteal/pull/205/commits/d5fa081c1635cb10055dfbbc82cc027428502226.

Reply:
looks good!